### DISCUSSION TITLE
Add download button on homepage of website

### DISCUSSION THREAD

**Discussion by UnixPhonez** [stoatchat/discussions#395] [Feature Suggestions]
I saw a reddit user comment to add a download button that automatically detects an users operating system to the homepage of https://revolt.chat/ and that is quite a good idea. As not everyone is tech savvy and knows how to navigate an github repo

**Comment by 4444dogs** [comment]
I agree because when I first signed up for Revolt, it took me forever to figure out where to download the desktop version.

**Comment by Schweeeeeeeeeeeeeeee** [comment]
Definitely a needed feature

**Comment by amytimed** [comment]
Yeah, needing to go on the GitHub repo is kinda annoying sometimes

**Comment by insertish** [comment]
The website is being replaced in the near future and will feature a download button + page. (It's almost ready just a few pages need to be rewritten)
